### PR TITLE
Tweaks HoP job disk selling vendor

### PR DESCRIPTION
**Pull request by EvilDragonfiend**
## About The Pull Request
Tweaks HoP job disk selling vendor

## Why It's Good For The Game
some job disks become unavailable to regain once these are lost (i.e. PDA is exploded or broken, or something else)
This PR makes job disks are not precious more than how these currently are.

## Testing Photographs and Procedure

<details>
<summary>Screenshots&Videos</summary>

image

</details>

## Changelog
:cl:
code: relocated cartridge.dm to job_disk.dm. repathed cartridge vendor to /job_disk
tweak: HoP disk vendor sells various job disks now. there're contraband inside.
tweak: HoP disk vendor no longer sells PDAs that have weirdly-pre-inserted-head-only-job-disks. It sells normal PDAs now.
/:cl:

### CODE REVIEW

File: code/modules/vending/job_disk.dm
Comment (Archanial) [inline code/modules/vending/job_disk.dm]: change the name to reflect change in filename

File: code/modules/vending/job_disk.dm
Comment (Archanial) [inline code/modules/vending/job_disk.dm]: what is the difference between those two? Do you need both or is the toxin one enough?

File: code/modules/vending/job_disk.dm
Comment (Archanial) [inline code/modules/vending/job_disk.dm]:
```suggestion
					/obj/item/computer_hardware/hard_drive/role/security = 3,
```

File: code/modules/vending/job_disk.dm
Comment (Archanial) [inline code/modules/vending/job_disk.dm]:
```suggestion
					/obj/item/computer_hardware/hard_drive/role/roboticist = 3,
```

File: code/modules/vending/job_disk.dm
Comment (EvilDragonfiend) [reply code/modules/vending/job_disk.dm]: signal: only has signal program
toxins: signal + atmos scan + chem scan.
actually pathing is weird.

Comment (Archanial) [reply code/modules/vending/job_disk.dm]: remove normal signal and leave toxins I guess, you don't need them doubled

**Review by Archanial — DISMISSED** [review]

**Comment by itsmeow** [comment]
Something else I wanted to do was add PDA casings to the silicate selections vendor and laptop/tablet builder, not sure if you'd want to try that separately

File: code/modules/vending/job_disk.dm
Comment (itsmeow) [inline code/modules/vending/job_disk.dm]: might as well rename this to job_disk as well

**Review by itsmeow — CHANGES_REQUESTED** [review]

File: code/modules/vending/job_disk.dm
Comment (EvilDragonfiend) [reply code/modules/vending/job_disk.dm]: Heh I overlooked it

**Comment by EvilDragonfiend** [comment]
> Something else I wanted to do was add PDA casings to the silicate selections vendor and laptop/tablet builder, not sure if you'd want to try that separately

Well that's a good idea but I want to finish my things first

**Comment by EvilDragonfiend** [comment]
chamged

**Review by PestoVerde322 — APPROVED** [review]
Mapping and codewise seems fine, approved on the mapside, wait on approval on code.

**Review by itsmeow — APPROVED** [review]

**Pull request merged**
